plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetastoreAccessOperations.java

```diff
@@ -319,6 +321,26 @@ public void testUnregisterTable()
                         .build());
     }
 
+    @Test
+    public void testInformationSchemaColumns()
+    {
+        int tables = 3;
+        for (int i = 0; i < tables; i++) {
+            assertUpdate("CREATE TABLE test_select_i_s_columns" + i + "(id VARCHAR, age INT)");
+        }
+
+        assertMetastoreInvocations("SELECT * FROM information_schema.columns WHERE table_name LIKE 'test_select_i_s_columns%'",
+                ImmutableMultiset.builder()
+                        .add(GET_ALL_DATABASES)
+                        .add(GET_ALL_TABLES_FROM_DATABASE)
+                        .addCopies(GET_TABLE, 3)
```

Comment:
Small improvement might be to have one table that does not match the `LIKE` string and show that it does not need a `GET_TABLE` call.